templates/admin/notice.tmpl

```diff
@@ -97,4 +99,4 @@
 		<p></p>
 	</div>
 </div>
-{{template "base/footer" .}}
+{
```

Comment:
this should be changed back to `{{template "base/footer" .}}`

Reply:
Done @lafriks